### DISCUSSION TITLE
How i am use SPI to transmit Heltec lora32 v2 data to arduino ou serial for pc ?

### DISCUSSION THREAD

**Discussion by HelloWar75** [cyberman54/ESP32-Paxcounter#727] [Q&A] [answered]
How i am use SPI to transmit Heltec lora32 v2 data to arduino ou serial for pc ?

**Comment by cyberman54** [comment, marked as answer]
You need a local host which acts as SPI master.  Could be a Arduino or PC. You need to write a small program to receive and process the data.

**Reply by HelloWar75** [reply]
Ok, I found only how to activate SPI for SD CARD, but for arduino or pc where would it be?
